test/src/Makefile

```diff
@@ -7,6 +7,7 @@ PATH 	:= $(PATH):$(GOBIN)
 
```

Comment:
please update this Makefile, use the content from https://github.com/cloudposse/terraform-aws-service-control-policies/blob/master/test/src/Makefile